docs/_sass/components/docs/_doc-content-learn-by-example.scss

```diff
@@ -0,0 +1,52 @@
+// DOC CONTENT CORE
+// -----------------------------------------------
+// -----------------------------------------------
+
+.doc-content-learn-by-example {
```

Comment:
@israelperezglez should this CSS live here or should it be elsewhere?

Reply:
We don't have the definitive style for this new section in the website. We just copied the styles from Arrow Core. It's just a way to preview this new section in the website. We'll change it as soon as we have the new styles :+1:

Reply:
It happens the same with #32. We're focused on the content while styles are being built by design team.

Reply:
Design is expected to work over this PR to polish / update design with a new logo and colors. They can decide where to put css at that point, but this is like any other arrow docs sites, their styles already within this repository, which kind of acts like a coordination one.

Reply:
@raulraja Yeah, we could use that scss ;)